e2e_samples/parking_sensors_synapse/README.md

```diff
@@ -163,7 +163,6 @@ Please check the details [here](docs/observability.md).
 
 ## How to use the sample
 
-<!--TODO: update this to run for synapse sample-->
```

Comment:
On line 128, the link is broken. The image file name is "CI_CD_process_sequence.png".

Reply:
fixed